modules/ROOT/pages/install-patches.adoc

```diff
@@ -0,0 +1,111 @@
+= Perform Scheduled Maintenance
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Use the following procedure when performing scheduled maintenance for Anypoint Runtime Fabric nodes, such as applying OS patches and security updates. 
+
+== Controller Node Maintenance
+
+The controller nodes are the entry points into your Runtime Fabric cluster, but do not run any application workloads.
+
+To update controller nodes:
+
+. On your upstream TCP load balancer, remove the controller nodes from your load balancing pool, one at a time, during patching and rebooting tasks. 
+. When updates are complete on each controller node, add the node back to the cluster before removing the next one.
+
+== Worker Node Maintenance
+
+To update worker nodes: 
+
+. On a controller node, run the `gravity shell` command.
+. Verify that the status for all nodes is `READY`:
++
+```
+$ kubectl get nodes
+
+NAME          STATUS   ROLES    AGE   VERSION
+172.31.0.9    Ready    node     1d   v1.x.x
+172.31.0.8    Ready    master   1d   v1.x.x
+```
+. Retrieve the application namespace (environment ID): 
++
+```
+$ sudo kubectl get ns
```

Comment:
btw just in general once we enter the gravity shell, we don't need to prefix sudo in any of the subsequent kubectl commands, since the user it root already.
```
ip-172-31-0-91:/$ whoami
root
ip-172-31-0-91:/$
```

Reply:
Ah - DUH on my part. :) I've removed the subsequent occurrences of `sudo`. Thank you!